packages/cli/CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 # Change Log
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Added `--data-set-type` flag to create sequential data set command to allow for creating extended and large formatted sequential data sets [#2141](https://github.com/zowe/zowe-cli/issues/2141)
```

Comment:
- Add the particle: "Added **the** `bbbb` flag ... "
- Should "command" be plural?: "... sequential data set **commands** to allow ..."
- Your sentence needs to end w/ a period.